api/app/src/main/kotlin/packit/security/WebSecurityConfig.kt

```diff
@@ -58,6 +58,20 @@ class WebSecurityConfig(
 
     @Bean
     @Order(2)
+    fun publicEndpointSecurityFilterChain(httpSecurity: HttpSecurity): SecurityFilterChain {
```

Comment:
This is kind of off topic, but I wish we didn't have to list endpoints that need authorization or not in this file. The lack of `@PreAuthorize` should be enough really. I wonder if Spring has anything for this.